src/styles/main.scss

```diff
@@ -1,3 +1,262 @@
+  box-shadow: 0 2px 4px rgba(0,0,0,0.05);
+
+  &__logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    color: $main-accent;
+
+    transition: 0.3s;
+
+    &:hover {
+      color: $blue-accent;
+    }
+  }
+
+  &__item {
+    display: block;
+    line-height: 60px;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.is-active {
+  color: $blue-accent;
+  position: relative;
+}
+
+.is-active:after {
+  content: "";
+
+  display: block;
+  position: absolute;
+  bottom: 0;
+
+  width: 100%;
+  height: 4px;
+
+  background-color: $blue-accent;
+  border-radius: 8px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  border: 1px solid $gray-elements;
+  border-radius: 5px;
+
+  font-size: 10px;
+  line-height: 14px;
+  padding: 32px 16px 16px;
+
+  color: $main-accent;
+  background-color: $white;
+
+  margin: 0;
+
+  transition: transform 300ms ease-in-out;
+
+  &__image {
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+
+    &-img {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__product-title {
+    margin: 0 0 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: $big_line-height;
+  }
+
+  &__product-code {
+    margin: 0 0 16px;
+    color: $secondary;
+  }
+
+  &__review {
+    margin: 0;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review-text {
+    margin: 0;
+    color: $main-accent;
+    padding: 0;
+  }
+
+  &__price {
+    margin: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-value {
+      margin: 0;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: $big_line-height;
+    }
+
+    &-text {
+      margin: 0;
+      font-size: 12px;
+      line-height: $big_line-height;
+    }
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    background-color: $blue-accent;
+    border: 1px solid $blue-accent;
+
+    text-transform: uppercase;
+    text-decoration: none;
+    padding: 12px 70px;
+
+    font-weight: 700;
+    font-size: 14px;
+    border-radius: 5px;
+    line-height: 16px;
+    color: $white;
+
+    transition: 300ms ease-in-out;
+  }
+
+  &__link:hover {
+    background-color: $white;
+    border: 1px solid $blue-accent;
+    color: $blue-accent;
+  }
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(1.1);
+  }
+
+  &:hover &__product-title {
+    color: #34568b;
+  }
+}
+
+.stars {
+  display: flex;
+
+  &--1 :first-child,
+  &--2 :nth-child(-n+2),
+  &--3 :nth-child(-n+3),
+  &--4 :nth-child(-n+4),
+  &--5 :nth-child(-n+5) {
+    background: url(../images/star-active.svg) no-repeat;
+  }
```

Comment:
Try using for loops for the stars selector (Follow [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 12-th point )